Gruntfile.js

```diff
@@ -51,8 +51,7 @@ module.exports = function(grunt) {
     // The gen-importer tasks generates files that import dynamic paths read from
     // superdesk.config.js
     grunt.registerTask('ngtemplates:gen-importer', [
```

Comment:
Since this isn't grouping anything anymore, can we just remove this extra task and instead use `ngtemplates:gen-apps` everywhere instead of this `ngtemplates:gen-importer`?